docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```
@@ -942,3 +942,88 @@ spec:
   purpose: Informational
   tags: compliance, hunting
   contributors: nabilschear
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Windows Firewall, Domain Profile enabled (Windows)
+  query: SELECT 1 FROM registry WHERE path LIKE 'HKEY_LOCAL_MACHINE\Software\Policies\Microsoft\WindowsFirewall\DomainProfile\EnableFirewall' AND CAST(data as integer) = 1;
+  description: "Checks for the existence of a Group Policy that enables the Domain Profile for Windows Defender Firewall."
```

```suggestion
  description: "Checks if the domain firewall profile is enabled. The domain profile applies to networks where the host system can authenticate to a domain controller."
```

@defensivedepth is this accurate? Can we simplify this to and say that the policy checks if the domain firewall profile is enabled?

---

It doesn't though. According to CIS, the way you audit this is by checking to make sure that the GPO is enabled. The GPO enforces the policy that enables the firewall - the firewall could certainly be enabled without the GPO, but it can be easily disabled by a non-admin user. The GPO makes it more difficult for a user to disable it.

---

> According to CIS, the way you audit this is by checking to make sure that the GPO is enabled.

@defensivedepth aha, got it. I added this to my suggested changes.

I'm curious, does this mean that an auditor using CIS would not accept these settings being enforced by an MDM solution (ex. Miradore) instead of a Group Policy? Or, can MDM solutions not configure this setting?

My current understanding is that a Group Policy and a setting enforced by MDM are separate concepts.

---

> does this mean that an auditor using CIS would not accept these settings being enforced by an MDM solution (ex. Miradore) instead of a Group Policy?

Guillaume: Let’s just say, an auditor with a risk based approach would realize that the goal of the requirement is met, but there are some verrrry by the book ones who don’t care about the “intent of the law.”